src/http/http_rpc_context.h

```diff
@@ -100,21 +100,6 @@ namespace http
           auto& signed_headers = parsed_sign_params->signed_headers;
           signed_headers.emplace_back(http::headers::AUTHORIZATION);
 
-          for (const auto& required_header :
```

Comment:
This is during canonicalisation. We no longer mandate an odd set of required signed headers here. We will canonicalise anything, keeping the signed headers. Later, when we verify the signature in `http_sig,h`, we check for some required headers. These are currently static, in `http_consts.h`. There may be a use case for making this choice endpoint-specific, but we haven't seen it yet.